examples/05_Collections/03_Map.md

```diff
@@ -0,0 +1,47 @@
+# Map
+
+A [Map](https://kotlinlang.org/docs/reference/collections.html) is collection of key/value pairs, where each key is unique and is used to retrieve the corresponding value. Like Lists and Sets, you can create a mutable map with `mutableMapOf()`, an immutable map with `mapOf()` and a readonly view of a mutable map with the [Map&lt;K, V&gt;](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-map/index.html) interface. 
```

Comment:
The second sentence should be:
"For creating maps, there are functions `mapOf()` and `mutableMapOf()`. A read-only view of a mutable map can be obtained by casting it to `Map`."